includes/forms/class-llms-form-field.php

```diff
@@ -784,21 +785,47 @@ protected function prepare_password_strength_meter() {
 		 */
 		$meter_settings = apply_filters( 'llms_password_strength_meter_settings', $meter_settings, $this->settings, $this );
 
-		// If scripts have been enqueued, add password strength meter script and localize with meter data.
+		// If scripts have been enqueued, add password strength meter script.
 		if ( did_action( 'wp_enqueue_scripts' ) ) {
-
-			wp_enqueue_script( 'password-strength-meter' );
-			llms()->assets->enqueue_inline(
-				'llms-pw-strength-settings',
-				'window.LLMS.PasswordStrength = window.LLMS.PasswordStrength || {};window.LLMS.PasswordStrength.get_settings = function() { return JSON.parse( \'' . wp_json_encode( $meter_settings ) . '\' ); };',
-				'footer',
-				15
+			$this->enqueue_strength_meter( $meter_settings );
+		} else { // Otherwise add it whe `wp_enqueue_scripts` is fired.
+			add_action(
+				'wp_enqueue_scripts',
+				function() use ( $meter_settings ) {
+					$this->enqueue_strength_meter( $meter_settings );
+				}
 			);
-
 		}
 
 	}
 
+	/**
+	 * Enqueue password strength meter script.
+	 *
+	 * @since [version]
+	 *
+	 * @param array $meter_settings {
+	 *     Hash of meter configuration options.
+	 *
+	 *     @type string[] $blocklist    A list of strings that are penalized when used in the password. See "user_inputs" at https://github.com/dropbox/zxcvbn#usage.
+	 *     @type string   $min_strength The minimum acceptable password strength. Accepts "strong", "medium", or "weak". Default: "strong".
+	 *     @type int      $min_length   The minimum acceptable password length. Must be >= 6. Default: 6.
+	 * }
+	 * @return void
+	 */
+	public function enqueue_strength_meter( $meter_settings ) {
```

Comment:
```suggestion
	private function enqueue_strength_meter( $meter_settings ) {
```

Sometimes you disgust me.

Reply:
oh yeah that's because at first I thought at that as a hook callback <3

Reply:
Yea i figured